sdk/identity/azure-identity/azure/identity/_credentials/chained.py

```diff
@@ -61,16 +65,26 @@ def get_token(self, *scopes, **kwargs):  # pylint:disable=unused-argument
         for credential in self.credentials:
             try:
                 token = credential.get_token(*scopes, **kwargs)
+                _LOGGER.info("%s acquired a token from %s", self.__class__.__name__, credential.__class__.__name__)
                 self._successful_credential = credential
                 return token
             except CredentialUnavailableError as ex:
                 # credential didn't attempt authentication because it lacks required data or state -> continue
                 history.append((credential, ex.message))
+                _LOGGER.info("%s - %s is unavailable", self.__class__.__name__, credential.__class__.__name__)
             except Exception as ex:  # pylint: disable=broad-except
                 # credential failed to authenticate, or something unexpectedly raised -> break
                 history.append((credential, str(ex)))
+                _LOGGER.warning(
+                    '%s.get_token failed: %s raised unexpected error "%s"',
```

Comment:
Given this is chained credential, should this be a warning? Or just info?

Reply:
The design guidelines stipulate exceptions be logged as warnings.